docs/setup/settings.asciidoc

```diff
@@ -565,7 +565,8 @@ all http requests to https over the port configured as <<server-port, `server.po
 
 [[server-ssl-supportedProtocols]] `server.ssl.supportedProtocols`::
 An array of supported protocols with versions.
-Valid protocols: `TLSv1`, `TLSv1.1`, `TLSv1.2`, `TLSv1.3`. *Default: TLSv1.1, TLSv1.2, TLSv1.3*
+Valid protocols: `TLSv1`, `TLSv1.1`, `TLSv1.2`, `TLSv1.3`. *Default: TLSv1.2, TLSv1.3*
+Enabling `TLSv1.1` would require both setting the `--tls-min-1.1` option in the `node.options` configuration and adding `TLSv1.1` to `server.ssl.supportedProtocols`.
```

Comment:
nit: should we mention anything about the `HTTP/2` requirement of the minimum TLSv1.2?

Reply:
added in 2d215ed97fbf810b0abc410d73e75ff848a0c10b